### DISCUSSION TITLE
copy node and paste it as 'Pointer' node

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/freeplane#1052] [Ideas]
Hi,
I was creating a new feature request to have a new command to paste a node as a clone (without descendants), BUT having a link to the original node.

Something similar to a clone but not a clone. A pasted node that looks always like the original, but can't be edited and directs you to the original node if you want (like a reference)

Finally I ended creating this script to create a "pointer" node from the selected one and then I can move it were I want:

```groovy
def n = node

def pointer = n.createChild()

pointer.link.node = n
pointer.text    = '=node.link.node?.text?:\'\''
pointer.details = '=node.link.node?.details?:\'\''
pointer.note    = '=node.link.node?.note?:\'\''


```
It's not what I originally wanted, but it's good enough.
hope it helps,

edo

**Comment by macmarrum** [comment]
Great minds think alike :wink: 

I've been using a similar script – [pasteAsSymlink.groovy](https://raw.githubusercontent.com/macmarrum/freeplane-scripting/main/src/scripts/pasteAsSymlink.groovy)
It pastes one or more copied nodes, each as a "symbolic link". 
The formula to be used can be optionally specified as an attribute on the root node
- root['pasteAsSymlinkText']
- root['pasteAsSymlinkDetails']
- root['pasteAsSymlinkNote']

There's also a companion script – [convertSymlinkToMaterializedsymlink.groovy](https://raw.githubusercontent.com/macmarrum/freeplane-scripting/main/src/scripts/convertSymlinkToMaterializedsymlink.groovy). It "materializes" the symlink back into a static node, keeping the original node reference and the formulas as attributes, so that it can be restored as a symlink, if needed.

**Reply by EdoFro** [reply]
great, thanks!

**Comment by Illegit** [comment]
Hi Sirs, may you explain with more details what is the aim of the original request of this thread, because it seems to me that the solution already exists, but eventually I'm interested to utilize other ways to manage clones and their copy in my maps.
the current apparently existing solution is copy only the node (menu: edit/copy/copy node (single)) and paste the clone (menu: edit/paste clone), what is the difference of what you are looking for?
Thank you

**Comment by Illegit** [comment]
And to jump to previous clones I use the script jump to next clone
[JumpToNextClone.zip](https://github.com/freeplane/freeplane/files/11069712/JumpToNextClone.zip)
Pay attention that this script sometimes doesn't work correctly with 2 topics:
1) doesn't jump (often it seems related to the map slow motion status)
2) doesn't center the next clone after jumping
